app/assets/javascripts/calendar.es6

```diff
@@ -11,6 +11,21 @@ class Calendar {
     const defaultConfig = {
       allDaySlot: false,
       cookieName: this.$el.attr('id') || 'calendar',
+      customButtons: {
+        jumpToDate: {
+          text: 'Jump to date',
+          click: (event) => {
```

Comment:
I know we don't have tests yet but if this was moved into it's own named method it'll make it easier to test & debug when we do.